modules/pam_access/pam_access.c

```diff
@@ -739,7 +744,7 @@ network_netmask_match (pam_handle_t *pamh,
 	*netmask_ptr = 0;
 	netmask_ptr++;
 
-	if (isipaddr(tok, &addr_type, NULL) == NO)
+	if (isipaddr(tok, &addr_type, &tok_addr) == NO)
```

Comment:
`tok_addr` seems to be unused after this initialization, I wonder what was the intent of this change.

Reply:
Correct, seems to come from an old version of the patch, reverted.